pyfftw/pyfftw.pyx

```diff
@@ -1408,6 +1436,11 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho is None:
+            ortho = self._ortho
+        if normalise_idft is None:
+            normalise_idft = self._normalise_idft
+
         if ortho and normalise_idft:
             raise ValueError('Invalid options: ortho and normalise_idft cannot'
                              ' both be True.')
```

Comment:
It's line 1444 which is the problem, not the lines above. Basically, lines 1444-6 need tweaking to match the new signature.